Src/BootCamp.Chapter/LogUtility/Logger.cs

```diff
@@ -0,0 +1,26 @@
+﻿using BootCamp.Chapter.LogUtility;
+using System.IO;
+
+namespace BootCamp.Chapter
+{
+    public class Logger
+    {
+        private const string LogPath = @"..\..\..\Output\Log.txt";
+
+        public Logger()
+        {
+            File.Delete(LogPath);
+        }
+
+        private ILogger SetLogOption()
+        {
+            return new ConsoleLogger();
+            //return new FileLogger(LogPath);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this commented out code.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5147695)